docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -44,6 +43,8 @@ security settings from your {es} cluster, authenticates to {es} with the
 . Install the appropriate https://docs.docker.com/get-docker/[Docker application]
 for your operating system.
 
+. If you're running on Linux, install https://docs.docker.com/compose/install/[Docker Compose].
```

Comment:
This should actually be in the Docker Compose Prerequisites section, not here (Docker).

Reply:
++ I'll move that line to the Docker Compose prerequisites.